webapp/src/ts/services/xml-forms.service.ts

```diff
@@ -283,7 +297,10 @@ export class XmlFormsService {
           .then(xml => ({ doc, xml }))
           .catch(err => {
             if (err.status === 404) {
-              return Promise.reject(new Error(`The form "${internalId}" doesn't have an xform attachment`));
+              const errorMessage = `The form "${internalId}" doesn't have an xform attachment`;
+              console.error('Error in XMLFormService getDocAndFormAttachment : ', errorMessage);
+              this.feedbackService.submit('Error in XMLFormService getDocAndFormAttachment : ' + errorMessage, false);
+              return Promise.reject(new Error(errorMessage));
             }
             throw err;
```

Comment:
I think a feedback doc will need to be created when any error is thrown, not just 404s.

Reply:
ok